src/o365/teams/commands/message/teams-message-reply-list.ts

```diff
@@ -0,0 +1,123 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import Utils from '../../../../Utils';
+import { Reply } from '../../Reply';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  teamId: string;
+  channelId: string;
+  messageId: string;
+}
+
+class TeamsMessageReplyListCommand extends GraphItemsListCommand<Reply>  {
+  public get name(): string {
+    return `${commands.TEAMS_MESSAGE_REPLY_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Retrieves a message from a channel in a Microsoft Teams team';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const endpoint: string = `${this.resource}/beta/teams/${args.options.teamId}/channels/${args.options.channelId}/messages/${args.options.messageId}/replies`;
+
+    this
+    .getAllItems(endpoint, cmd, true)
+    .then((): void => {
+        if (args.options.output === 'json') {
+          cmd.log(this.items);
+        }
+        else {
+          cmd.log(this.items.map(m => {
+            return {
+              id: m.id,
+              body: m.body.content
+            }
+          }));
+        }
+        
+        if (this.verbose) {
+          cmd.log(vorpal.chalk.green('DONE'));
+        }
+        cb();
+    }, (err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-i, --teamId <teamId>',
+        description: 'The ID of the team where the channel is located'
+      },
+      {
+        option: '-c, --channelId <channelId>',
+        description: 'The ID of the channel that contains the message'
+      },
+      {
+        option: '-m, --messageId <messageId>',
+        description: 'The ID of the message to retrieve'
```

Comment:
Description doesn't match the spec and the description in the .md file

Reply:
✔